File: 04.动静态库使用/03.静态库冲突/ZSApp/ZSApp/ViewController.m
//
//  ViewController.m
//  ZSApp
//
//  Created by zhoushuai on 2021/10/24.
//

#import "ViewController.h"
#import <ZSTools.h>
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    ZSTools *tools = [ZSTools new];
    [tools testPrint:nil];
}

@end

File: 06.Swift库与总结/完成代码/05.iOS混编代码规范/SwiftApp/SwiftApp/ViewController.swift
//
//  ViewController.swift
//  SwiftApp
//
//  Created by zhoushuai on 2022/5/29.
//

import UIKit
import OCFramework

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()

        let tool = ZSOCTool()

        // 使用宏定义前：
        //let weather:Weather = .rain
        //let psName:PersonName = "ZS"
        //let fineName = tool.findName(for: 0)
        

        // 使用宏定义后：
        let weather:Weather = .rain
        let psName:PersonName = PersonName(rawValue: "ZS")
        let fineName = tool.findName(forIndex: 0)
        
        print("wealth:\(weather.rawValue) psName:\(psName) finName:\(fineName)")
        

    }
}
